Wikipedia/Code/ArticleViewController.swift

```diff
@@ -459,6 +460,7 @@ class ArticleViewController: ViewController, HintPresenting {
             return
         }
         showAnnouncementIfNeeded()
```

Comment:
Careful with this area - this line will call the fundraising banner. I'm sure we don't want the YiR feature announcement to mess with it's presentation. You should only present YiR feature announcement if a fundraising banner will not present, IMO (be sure to confirm that with Product).